pages/apis/rest_api/analytics/_runs_list_query_strings.md

```diff
@@ -0,0 +1,12 @@
+<table>
+<tbody>
+  <tr>
+    <th>
+      <code>build_id</code>
+    </th>
+    <td>
+      <span>Filters the results by the given build UUID.</span>
```

Comment:
Nit: I noticed the [builds API docs has examples](https://buildkite.com/docs/apis/rest-api/builds)

```suggestion
      <span>Filters the results by the given build UUID.</span>
      <p class="Docs__api-param-eg"><em>Example:</em> <code>?build_id=018c133d-5419-7fe3-9e9e-3c51464490a2</code></p>
```